src/lib/stack.js

```diff
@@ -61,7 +61,13 @@ function getDiskQuota () {
 
     return res.json()
   })
-  .then(json => parseInt(json.data.attributes.quota, 10))
+  .then(json => {
+    if (Number.isInteger(json.data.attributes.quota)) {
+      return parseInt(json.data.attributes.quota, 10)
+    } else {
+      return 100000000000 // @TODO Waiting for instructions about how to deal with limitless instances
```

Comment:
I get your point but is it really relevant giving the fact that there may not be any default value for limitless instance? This chunk of code is only there to avoid breaking the view, besides that it's totally trashable.
Not that i don't wanna do it, it's just that i don't understand the need for clean trash code.